### PR TITLE
Add itk-rtk

### PR DESCRIPTION
**Pull request by SimonRit**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by SimonRit** [comment]
@scopatz, I created this based on @thewtex's #8838 for another ITK module, RTK. I seem to have the same problem with the python version. Any suggestion on how I should proceed from here? Thanks in advance!

**Comment by SimonRit** [comment]
@conda-forge/help-python It's not clear what the status of my PR. Is here anything I should do to finalize it? Thanks for your help.

**Comment by scopatz** [comment]
Hi @SimonRit - sorry for the delay in review.  I was on vacation

File: recipes/itk-rtk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  sha256: a7b2e40f3567c1c00a821413033f5531ed4490c8dc8f34182911520f1767e0c6  # [linux and py27]
+  sha256: 1484826ae76f8b66739b44ae4f947c52262c604480f11caafdbe73760de538bc  # [linux and py36]
+  sha256: 50281724f538941b268cee949c2877642027c2517dea2983be82e1381b9356ad  # [linux and py37]
+
+  sha256: 2cdb43a3666dd6ce4a8784b1c9284e515ac112c58263548ff6fb20fe2ebbb340  # [win and py36]
+  sha256: 559f5cb5b201c63623abee074fd79fa08c47f6deba72227b86d98109cdb81b49  # [win and py37]
+
+build:
+  number: 0
+  skip: True
```
Comment (scopatz) [inline recipes/itk-rtk/meta.yaml]: This will not create any packages because you are always skipping.  This line should probably be removed.

**Comment by scopatz** [comment]
I am nit a huge fan of creatin g packages from wheels. Is there any way to build from source?

**Comment by SimonRit** [comment]
Thanks for the review! Sorry, I wrongly modified #8838 on which this is based, it's been corrected now.

> I am nit a huge fan of creatin g packages from wheels. Is there any way to build from source?

I do build the wheels so I guess we could build from source. But for the wheels, I'm actually using [ITKPythonBuilds](https://github.com/InsightSoftwareConsortium/ITKPythonBuilds) which are scripts developed by ITK (mainly @thewtex I guess) and that would probably be a big effort to do the build here. @thewtex, what's your thoughts since since the same applies to itk-meshtopolydata?

**Comment by thewtex** [comment]
Eventually, we might be able to build from source. But, there other large hurtles that need to be cleared, e.g. building `itk` from source on conda-forge. I do not think they should hold up making this package available on conda-forge, though.

**Comment by SimonRit** [comment]
@scopatz, given @thewtex answer, do you agree on this PR or should I give up the idea of having an itk-rtk recipe (and close this PR)?

**Comment by scopatz** [comment]
I am not blocking this package :) @conda-forge/staged-recipes might have someone else who is more interested in merging it

**Comment by SimonRit** [comment]
Oops sorry and thanks. It's not clear to me what's the procedure to finalize this.

**Comment by isuruf** [comment]
Can you explain in more detail what the issues are in building from source?

**Comment by isuruf** [comment]
Also note that the `itk` package on conda-forge is violating the redistribution terms of its dependencies. https://github.com/conda-forge/itk-feedstock/issues/11

Is this recipe doing the same?

**Comment by SimonRit** [comment]
To build itk-rtk wheels from source, I use the [ITKPythonPackage](https://github.com/InsightSoftwareConsortium/ITKPythonPackage). I don't think I would be able to convert this to a conda recipe but for specific issues, I guess @thewtex could answer.
I am not aware of redistribution violations in itk-rtk but it relies on itk.

**Comment by SimonRit** [comment]
It's not clear to me what is the status of this PR. Who should act to finalize it or close it? Thanks in advance!

**Comment by scopatz** [comment]
It should be totally fine to use `ITKPythonPackage` on conda-forge and then use pip here to install the wheel that is created. I think that would work.

**Comment by SimonRit** [comment]
Ok thanks. Sure I can do that, but it was my understanding that it was preferable to avoid mixing conda and pip. I don't know what's the advantage of installing itk via conda in the first place if I start using pip.
I also don't understand why this recipe is blocked but similar ones as itk-meshtopolydata are accepted. Anyway, do you want me to close this PR?

**Comment by isuruf** [comment]
> Sure I can do that, but it was my understanding that it was preferable to avoid mixing conda and pip. I don't know what's the advantage of installing itk via conda in the first place if I start using pip.

@scopatz was mentioning that we build the wheel in the recipe and install it instead of just downloading the wheel and installing it.

> I also don't understand why this recipe is blocked but similar ones as itk-meshtopolydata are accepted. 

Because we didn't have the capacity to build the wheels from scratch at the time and now we have enough capacity and time in our CI to do it. Using pre-built wheels should be done as a last resort.

**Comment by SimonRit** [comment]
Thanks for the clarification, I had indeed misunderstood what @scopatz meant.
@thewtex, do you think that's possible with the current version of ITKPythonPackage? I remember pointing out a few things in [this post](https://github.com/InsightSoftwareConsortium/ITK/pull/477#issuecomment-462698900) and I wonder if can get around them in the CI.

**Comment by thewtex** [comment]
Now that conda-forge is using Azure Pipelines for CI, it may be possible... here is a PR to test viability with the *itk* package: https://github.com/conda-forge/itk-feedstock/pull/12

**Comment by thewtex** [comment]
bump

**Pull request closed without merging**